model_info/enwiki.nettrom_wp10.md

```diff
@@ -1,12 +1,12 @@
 Model Information:
 	 - type: GradientBoosting
 	 - version: 0.8.2
-	 - params: {'presort': 'auto', 'verbose': 0, 'warm_start': False, 'validation_fraction': 0.1, 'max_leaf_nodes': None, 'min_weight_fraction_leaf': 0.0, 'max_depth': 7, 'max_features': 'log2', 'init': None, 'min_samples_leaf': 1, 'population_rates': None, 'n_iter_no_change': None, 'min_impurity_split': None, 'criterion': 'friedman_mse', 'tol': 0.0001, 'min_samples_split': 2, 'n_estimators': 500, 'subsample': 1.0, 'loss': 'deviance', 'label_weights': None, 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'learning_rate': 0.01, 'random_state': None, 'scale': True, 'center': True, 'multilabel': False, 'min_impurity_decrease': 0.0}
+	 - params: {'presort': 'auto', 'n_iter_no_change': None, 'verbose': 0, 'random_state': None, 'learning_rate': 0.01, 'scale': True, 'min_samples_leaf': 1, 'min_weight_fraction_leaf': 0.0, 'loss': 'deviance', 'max_features': 'log2', 'tol': 0.0001, 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'center': True, 'subsample': 1.0, 'validation_fraction': 0.1, 'n_estimators': 500, 'criterion': 'friedman_mse', 'multilabel': False, 'init': None, 'max_leaf_nodes': None, 'min_samples_split': 2, 'population_rates': None, 'label_weights': None, 'min_impurity_split': None, 'warm_start': False, 'max_depth': 7, 'min_impurity_decrease': 0.0}
 	Environment:
-	 - revscoring_version: '2.6.8'
-	 - platform: 'Linux-4.9.0-11-amd64-x86_64-with-debian-9.12'
+	 - revscoring_version: '2.6.9'
```

Comment:
One of the commits mentions revscoring 2.8.1. Is this correct?

Reply:
It looks like it is not.  We should be using 2.8.2 actually.